app/models/formal_education.rb

```diff
@@ -0,0 +1,3 @@
+class FormalEducation < ApplicationRecord
+	validates :name, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
same as the other comment about changing the default behaviour of your editor -- 2 spaces instead of tab or 4 spaces